bears/java/JavaPMDBear.py

```diff
@@ -17,22 +17,14 @@ class JavaPMDBear:
     """
 
     LANGUAGES = {'Java'}
+    REQUIREMENTS = {MavenRequirement('net.sourceforge.pmd:pmd', '5.6.0'),
```

Comment:
Our CI test is using v 5.4.1 https://github.com/coala/coala-bears/blob/master/.ci/deps.sh#L163

But if you already check that it's not breaking then OK (and maybe we need to update the CI script)